### PR TITLE
Major & Minor refactoring + Format

### PR DESCRIPTION
**Pull request by Sigmanificient**
## Description

- Refactor `connect` & `handle_event` of ws_client
- Improve a few line of python elsewhere
- Reformat the code to black (can be dropped if needed)
- Update Makefile

*Cannot test due to circular import*

## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [x] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

File: EpikCord/abstract.py [hunk 1]
```diff
@@ -138,8 +131,8 @@ async def typing(self) -> TypingContextManager:
 
 
 class BaseCommand:
-    def __init__(self, checks: List[Check] = []):
-        self.checks: List[Check] = checks
+    def __init__(self, checks: List[Check] = None):
```
Comment (TheUntraceable) [inline EpikCord/abstract.py:134]: Shouldn't you mark this as Optional then?

Comment (Sigmanificient) [reply EpikCord/abstract.py:134]: indeed

File: EpikCord/close_event_codes.py [hunk 2]
```diff
@@ -4,6 +4,8 @@
 class GatewayCECode(IntEnum):
     """Gateway close event code."""
 
+    WEBSOCKET_ERROR = 1006
```
Comment (TheUntraceable) [inline EpikCord/close_event_codes.py:7]: The proper name for this is `UNUSUAL CLOSURE` or somethin,

**Review by TheUntraceable — CHANGES_REQUESTED** [review]
After you google the name of the code and replace it, I shall merge.

File: EpikCord/close_event_codes.py (hunk 2 above)
Comment (Sigmanificient) [reply EpikCord/close_event_codes.py]: *CLOSE_ABNORMAL: No close code frame has been receieved*

**Review by TheUntraceable — APPROVED** [review]

**Pull request merged**